src/browserbehavior/DefaultBrowserBehavior.ts

```diff
@@ -279,6 +279,10 @@ export default class DefaultBrowserBehavior implements BrowserBehavior, Extended
     return this.isAndroid();
   }
 
+  disable480pResolutionScaleDown(): boolean {
+    return /( Chrome\/98\.)/i.test(navigator.userAgent) && this.browser.os.startsWith('Windows');
```

Comment:
Is this bug reproducible in Chromium? If yes, have we tested this user agent in Chromium on Windows, e.g., Electron?